install-extensions.sh

```diff
@@ -29,7 +29,7 @@ function link_gcode_shell_command_extension()
 function link_settling_probe_command_extension()
 {
     echo "Linking extension to Klipper..."
-    ln -sf "${SRCDIR}/gcode_shell_command.py" "${KLIPPER_PATH}/klippy/extras/gcode_shell_command.py"
+    ln -sf "${SRCDIR}/settling_probe.py" "${KLIPPER_PATH}/klippy/extras/settling_probe.py"
```

Comment:
This commit is a perfect example of something that should have been squashed into a single commit. Also, it's a good example of a commit that needs it's commit message fixed.

Reply:
I found the mistakes and fixed them.  The last commit has this fixed.  I do not know how to "squish" a commit, do you want to teach me?